### ISSUE TITLE
Larger snippets fail

### ISSUE THREAD

**Issue by sjaakd** [jtulach/codesnippet4javadoc#1]
@jtulach : nice tool :+1: 

I'm trying to use it in our open source project were we use an annotation processor to generate bean-mappings (MapStruct). The javadoc examples would include samples of the generated code.

I've copied those samples (api + generated code) to the test directory of the api project. When I use too long snippets, the javadoc processor fails: `javadoc: error - Line is too long in: ../value/OrderMapperImpl.java`

Javadoc:
```java
/**
 * Configures the mapping of source constant value to target constant value.
 *
 * {@codesnippet OrderMapperImpl}
 *
 */
```
Targeted source code

```java
// BEGIN: OrderMapperImpl
public class OrderMapperImpl implements OrderMapper {

    @Override
    public ExternalOrderType orderTypeToExternalOrderType(OrderType orderType) {
        if ( orderType == null ) {
            return null;
        }

        ExternalOrderType externalOrderType_;

        switch ( orderType ) {
            case EXTRA: externalOrderType_ = ExternalOrderType.SPECIAL;
            break;
            case STANDARD: externalOrderType_ = ExternalOrderType.DEFAULT;
            break;
            case NORMAL: externalOrderType_ = ExternalOrderType.DEFAULT;
            break;
            case RETAIL: externalOrderType_ = ExternalOrderType.RETAIL;
            break;
            case B2B: externalOrderType_ = ExternalOrderType.B2B;
            break;
            default: throw new IllegalArgumentException( "Unexpected enum constant: " + orderType );
        }

        return externalOrderType_;
    }

}
// FINISH: OrderMapperImpl
```

Any idea how to fix this:

**Comment by sjaakd** [comment]
Ok.. I saw that the line length is limited to 80 chars.. Would it be possible to make that configurable? We use 120 char's..

**Comment by jtulach** [comment]
Thanks for trying my utility. Ok, I make the length configurable.
-jt
Dne 28. 2. 2016 18:12 napsal uživatel "Sjaak Derksen" <
notifications@github.com>:

**Comment by sjaakd** [comment]
I added a PR:( #2 ) yesterday for this. Have a look whether its usable. Thanks & best regards,
Sjaak

**Comment by jtulach** [comment]
Fixed by #2 and released as [0.8](http://repo1.maven.org/maven2/org/apidesign/javadoc/codesnippet-doclet/0.8/).
